packages/builder/src/builder.js

```diff
@@ -568,6 +572,7 @@ export default class Builder {
       watcher.on(event, listener)
     }
 
+    // TODO: due to fixes in chokidar this isnt used anymore and could be removed in Nuxt v3
```

Comment:
As this was internal/undocumented is it necessary to keep it?

Reply:
I would leave it in for now until we are sure the chokidar fix works for everybody (and not just me). Ofc we could always do a hotfix in that case, but having a config backup option is nice too I guess. I will keep track of this TODO and submit a PR to remove this code in a next release if nobody complains :)

Reply:
Looks like a good plan :) (However we also have the possibility to do a fast patch release in case of complain but makes sense keeping it for a while)